tests/testapp/tests/test_serialization.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.test import TestCase
+from facility_profile.models import Facility
+from morango.utils.syncing_utils import _syncing_models
+
+
+class SerializationTestCase(TestCase):
+
+    def setUp(self):
+        self.bob = Facility.objects.create(name="bob")
+        self.dob = Facility.objects.create(name="dob")
+        self.bob.id = self.bob.id.hex
+        self.dob.id = self.dob.id.hex
```

Comment:
Probably good for serialization to be able to handle this case (`id` being a UUID object) internally. I believe that can be achieved by properly definition of certain methods on the `Field` class -- we have other `UUIDField` instances in other places (e.g. Kolibri) that may do that better.